discord-bot/bot/extensions/guild_settings.py

```diff
@@ -78,7 +78,6 @@ async def log_channel(ctx: lightbulb.SlashContext) -> None:
 
     # if the bot's permissions for this channel don't contain SEND_MESSAGE
     # This will also filter out categories and voice channels
-    print(permissions_in(ch, own_member) & hikari.Permissions.SEND_MESSAGES)
```

Comment:
(you can use logger.debug() to add some debug-tracing ... sometimes there location where you inserted print statements for dev are actually neuralgic points and tracing them might be useful for other debugging too)

Reply:
that was left from an upstream mistake, it will be corrected when this gets merged.